marconi/test-lib/Gen/Marconi/Types.hs

```diff
@@ -0,0 +1,75 @@
+{-# LANGUAGE RecordWildCards #-}
+
+module Gen.Marconi.Types
+  ( genSlotNo
```

Comment:
These are the generators used in Utxo related tests. I'll move @koslambrou generators here as well and refactor the commonalities if we're OK with this approach

Reply:
:+1: